packages/notifications/push/amplify_push_notifications/test/amplify_push_noitfications_impl_test.dart

```diff
@@ -431,37 +431,13 @@ void main() {
         )..onNotificationReceivedInBackground(
             testGlobalCallbackFunction,
           );
-        await Future.delayed(const Duration(seconds: 3), () {});
+        await Future.delayed(const Duration(microseconds: 1), () {});
 
         expect(pref.containsKey(externalHandleKey), isTrue);
       },
     );
   });
 
-  test('should invoke the top-level or static external callback function',
```

Comment:
Removing this test as it's covered in the [flutter api test cases](https://github.com/aws-amplify/amplify-flutter/blob/fix/push-android-tests/packages/notifications/push/amplify_push_notifications/test/push_notification_flutter_api_test.dart#L71).